externals/install-dotnet.sh

```diff
@@ -310,6 +310,10 @@ get_machine_architecture() {
             echo "s390x"
             return 0
             ;;
+        ppc64le)
+            echo "ppc64le"
+            return 0
+            ;;
```

Comment:
it is not clear why security related PR includes adding new architectures

Reply:
It came from the the new `install-dotnet.sh` script maintained by MS. It was updated in order not to fail tests.